src/DashboardService.Models.Db/DbComment.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Db;
+
+public class DbComment
+{
+  public const string ToTable = "Comments";
+  
+  public Guid Id { get; set; }
+  public Guid TaskId { get; set; }
+  public Guid CreatedBy { get; set; }
+  
+  public string Content { get; set; }
+  
+  public DateTime CreatedAtUtc { get; set; }
```

Comment:
CreatedBy and CreatedAtUtc and IsActive goes together after the rest properties but before navigation properties followed by empty line

Reply:
++